andela/ah-scorpion
# [#159053989] enable liking and disliking articles

**What does this PR do?**
Enable users to like or dislike an article

**Description of tasks completed**
Added the `like` endpoint and the `dislike` endpoint with the appropriate views.

**How can this PR be manually tested?**

- clone this branch.
- go to the url of your favorite article and append /like e.g `api/v1/articles/training-dragons/like` and make a PUT request to like it.
- Repeating thie above step will unlike the article
- go to the url of an article and append /dislike e.g `api/v1/articles/training-dragons/dislike` and make a PUT request to dislike it.
- Repeating thie above step will un-dislike the article
- run tests using `python manage.py test`. All tests should pass.

**What are the relevant pivotal tracker stories?**
[#159053989](https://www.pivotaltracker.com/story/show/159053989)
